pygac/gac_klm.py

```diff
@@ -522,56 +527,73 @@ def get_lonlat(self):
         self.lons, self.lats = gtp.Gac_Lat_Lon_Interpolator(arr_lon, arr_lat)
         return self.lons, self.lats
 
+    def get_header_timestamp(self):
+        year = self.head['start_of_data_set_year']
```

Comment:
No, I need the additional independent timestamp from the header here, not the timestamp of the first scanline which is read in `get_times()`. These two timestamps are independent, so that in most cases only one of them is corrupted.